pkg/router/template/router.go

```diff
@@ -578,6 +584,16 @@ func hasRequiredEdgeCerts(cfg *ServiceAliasConfig) bool {
 	return false
 }
 
+// hasReencryptDestinationCACert checks whether a destination CA certificate has been provided.
+func hasReencryptDestinationCACert(cfg *ServiceAliasConfig) bool {
+	destCertKey := generateDestCertKey(cfg)
+	destCACert, ok := cfg.Certificates[destCertKey]
+	if ok && len(destCACert.Contents) > 0 {
```

Comment:
I am ok with that - did a copy-pasta of the function above it so it follows the same scheme as above.. I'll put a comment in here and change it as part of a cleanup.